### ISSUE TITLE
Add an option to select the default video quality

### ISSUE THREAD

**Issue by juanfra684** [tommyxchow/frosty#210]
Hi, before of describing the FR let me thank you for making the app. This is a quite better experience than the official app.

**Is your feature request related to a problem? Please describe.**
My Internet connection is not the best and the most of time I've enough with 480p/360p video resolution. When I use the official twitch overlay, it lets me to select the quality but has always "auto" as default. With the optimized overlay, I can't even select the resolution.

**Describe the solution you'd like**
An option to select a default video resolution.

**Describe alternatives you've considered**
There is not alternatives.

**Comment by LeirbagGab** [comment]
That's a really good thing to implement it would help me out a lot too

**Comment by Syntox94** [comment]
Yes please. Im not getting Full Quality for whatever reason. i need a button to manually select the highest quality like the default twitch player.
